cluster-http/src/main/scala/akka/management/cluster/ClusterDomainEventServerSentEventEncoder.scala

```diff
@@ -0,0 +1,125 @@
+          eventType = Some(eventType)
+        )
+      )
+
+    event match {
+      case memberEvent: ClusterEvent.MemberEvent =>
+        memberEvent match {
+          case ClusterEvent.MemberJoined(member) =>
+            sse("MemberJoined", JsObject("member" -> encode(member)))
+
+          case ClusterEvent.MemberWeaklyUp(member) =>
+            sse("MemberWeaklyUp", JsObject("member" -> encode(member)))
+
+          case ClusterEvent.MemberUp(member) =>
+            sse("MemberUp", JsObject("member" -> encode(member)))
+
+          case ClusterEvent.MemberLeft(member) =>
+            sse("MemberLeft", JsObject("member" -> encode(member)))
+
+          case ClusterEvent.MemberExited(member) =>
+            sse("MemberExited", JsObject("member" -> encode(member)))
+
+          case ClusterEvent.MemberDowned(member) =>
+            sse("MemberDowned", JsObject("member" -> encode(member)))
+
+          case ClusterEvent.MemberRemoved(member, previousStatus) =>
+            sse("MemberRemoved", JsObject("member" -> encode(member), "previousStatus" -> encode(previousStatus)))
+        }
+
+      case leaderChanged: ClusterEvent.LeaderChanged =>
+        leaderChanged.leader match {
+          case Some(address) =>
+            sse("LeaderChanged", JsObject("address" -> encode(address)))
+
+          case None =>
+            sse("LeaderChanged", JsObject.empty)
+        }
+
+      case roleLeaderChanged: ClusterEvent.RoleLeaderChanged =>
+        roleLeaderChanged.leader match {
+          case Some(address) =>
+            sse(
+              "RoleLeaderChanged",
+              JsObject("role" -> JsString(roleLeaderChanged.role), "address" -> encode(address))
+            )
+
+          case None =>
+            sse("RoleLeaderChanged", JsObject("role" -> JsString(roleLeaderChanged.role)))
+        }
+
+      case ClusterEvent.ClusterShuttingDown =>
+        sse("ClusterShuttingDown", JsObject.empty)
+
+      case reachabilityEvent: ClusterEvent.ReachabilityEvent =>
+        reachabilityEvent match {
+          case ClusterEvent.UnreachableMember(member) =>
+            sse("UnreachableMember", JsObject("member" -> encode(member)))
+
+          case ClusterEvent.ReachableMember(member) =>
+            sse("ReachableMember", JsObject("member" -> encode(member)))
+        }
+
+      case dataCenterReachabilityEvent: ClusterEvent.DataCenterReachabilityEvent =>
+        dataCenterReachabilityEvent match {
+          case ClusterEvent.UnreachableDataCenter(dataCenter) =>
+            sse("UnreachableDataCenter", JsObject("dataCenter" -> JsString(dataCenter)))
+          case ClusterEvent.ReachableDataCenter(dataCenter) =>
+            sse("ReachableDataCenter", JsObject("dataCenter" -> JsString(dataCenter)))
+        }
+
+      case _ =>
+        // @TODO these are either internal, or added but forgot to update this module
+        // @TODO how should we handle them? maybe leave this as FIXME for now?
```

Comment:
I guess our options are to either crash, silently ignore them, or log a message. Silently ignoring them might be fine...

Reply:
I think some more are internal, SeenChanged.
Not often added but we actually have one coming soon. Let's ignore them here.